README.md

```diff
@@ -107,6 +107,11 @@ This will get the current data from the Trello board and update the data file
 with the data from the current day. If there already was some data in the file
 for the same day it will be overridden.
 
+To override the Trello board from the `burndown-data-SPRINT_NUMBER.yaml` with another board
```

Comment:
@TheRealBro 

> @alexandergraul 
Don't forget to change README.md please 😃
if you add new "--options" to a software

I don't think this is needed in the README. In the README we have the basic usage, all the option are documented with detail in the [man page](https://github.com/openSUSE/trollolo/blob/master/man/trollolo.1.md).  And the `--board-id` option is already documented there. It is true that the documentation can be improved, but I don't want to overload the README.

@TheRealBro can you open an issue to improve the documentation of the project so we can discuss this there instead of doing it in @alexandergraul PR? :wink: 

In the meanwhile, @alexandergraul can you remove this please? I think it is not needed, and we are not documenting something like that here for any other command, so for consistency as well. :smile: